src/components/Buttons/index.js

```diff
@@ -12,12 +12,17 @@ import {
   StyledSaveButton,
   StyledPlayButton,
   StyledImgPlayButton,
-  
+  StyledCancelButton,
+  StyledTxtSaveButton,
+  StyledEditButton,
+  StyledImgEditButton,
+
 } from "./style";
 
-import  '../../assets/fonts/coolicons.css'
-import logoG from '../../assets/fonts/icons/btn_google_light_normal_ios.svg'
-import playArrow from '../../assets/fonts/icons/play-arrow.svg'
+import "../../assets/fonts/coolicons.css";
+import logoG from "../../assets/fonts/icons/btn_google_light_normal_ios.svg";
+import playArrow from "../../assets/fonts/icons/play-arrow.svg";
+import edit from "../../assets/fonts/icons/edit.svg";
```

Comment:
Sergio, 
You are importing the svg files but its better to use the icons as fonts. Please check the header component Mario did recently to see how you can implement it, once you see it I'm sure you can make it straight-forward.